shell.nix

```diff
@@ -56,6 +56,14 @@ mkDerivation {
     export NIX_PATH=nixpkgs=${nixpkgs}
     export NIX_PATH_LOCKED=1
     echo   NIX_PATH set to $NIX_PATH >&2
+
+    alias io-csl-path-commit='echo -n "Enter cardano-sl nix store path: "; read csl_path; nix-store --query --references $(nix-store --query --deriver $csl_path) | grep cardano-sl- | grep -v sl-core\\\|sl-db\\\|sl-infra\\\|sl-lrc\\\|sl-ssc\\\|sl-godtossing\\\|sl-txp\\\|sl-update | cut -c56-62'
+    alias git-rev='git log -n1 --pretty=oneline'
```

Comment:
This alias is a preference of style, for example, I use `tig`. I prefer we add those aliases in `.bashrc` and people can use them on the deployment machines.